Makefile

```diff
@@ -72,7 +72,8 @@ check: python-env
 .PHONY: misspell
 misspell:
 	go get github.com/client9/misspell
-	$(FIND) -name '*' -exec misspell -w {} \;
+	# Ignore Kibana files (.json)
+	$(FIND) -not -path "*.json" -name '*' -exec misspell -w {} \;
```

Comment:
Would it make sense to also add it to `make check` so it is also run with the automated tests?

Reply:
@simitt We decided in the past to not do it as we worried it would get into our way in things like the kibana issue above. But now seeing that I took all the changes from the script, I like the idea.

Reply:
It would be cool to have it as lint bot, i.e. leave comments on the PR without breaking the build.

Reply:
that would be pretty neat for all our linting. So instead of having to go to the CI log you would get comments in the PR for non fmt things.